org.cvfosammmm.Lemma.yml

```diff
@@ -0,0 +1,49 @@
+id: org.cvfosammmm.Lemma
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: lemma
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=host
+  - --env=PATH=/app/bin:/usr/bin:/app/bin/x86_64-linux
+  - --env=LC_ALL=C
+  - --talk-name=ca.desrt.dconf
```

Comment:
are you sure you need dconf or is this just cut & paste?

Reply:
the latter.